node/src/transition_frontier/sync/transition_frontier_sync_effects.rs

```diff
@@ -76,6 +76,9 @@ impl TransitionFrontierSyncAction {
                 if let Some(callback) = on_success {
                     store.dispatch_callback(callback.clone(), ());
                 }
+
+                let hash = best_tip.hash.clone();
+                store.dispatch(P2pNetworkPubsubAction::BroadcastAcceptedBlock { hash });
```

Comment:
this should not happen here right? We just triggered `BlockNextApplyInit` above, we don't know if it is going to succeed or not. Only after `BlocksNextApplySuccess` we know it succeeded. Also if `BlocksNextApplyError` is dispatched, then we must reject this message.